regolith/schemas.py

```diff
@@ -177,6 +177,26 @@
     ''',
         'email': '<b>scopatz</b> <i>(AT)</i> <b>cec.sc.edu</b>'
     },
+    'institutions': {'_id': 'columbiau',
+                     'aka': ['Columbia University', 'Columbia'],
+                     'city': 'New York',
+                     'country': 'USA',
+                     'departments':
+                         - {'chemistry': 'Department of Chemistry'}
```

Comment:
I think this is causing some of the travis failures. This is the correct list syntax for yaml, but not in python.